config/config.go

```diff
@@ -33,19 +33,23 @@ type (
 	}
 
 	Config struct {
-		Log  Log  `yaml:"log"`
-		Host Host `yaml:"host"`
-		Dev  Dev  `yaml:"dev"` // WARN: do not expose dev settings as flags
+		Log         Log  `yaml:"log"`
+		Host        Host `yaml:"host"`
+		Dev         Dev  `yaml:"dev"` // WARN: do not expose dev settings as flags
+		EnablePprof bool `yaml:"enable-pprof"`
```

Comment:
question: What do you think about
```
debug:
  enable-pprof: true #
  collectors:  # so that we group all debugging under debug
   - go
   - process
```